packages/monorepo-scripts/package.json

```diff
@@ -7,6 +7,8 @@
     "scripts": {
         "build:watch": "tsc -w",
         "deps_versions": "node ./lib/deps_versions.js",
+        "publish": "yarn build; node ./lib/publish.js",
+        "convert_changelogs": "yarn build; node ./lib/convert_changelogs.js",
```

Comment:
where are we running this?

Reply:
This script is temporary. We will use it once to migrate over to JSON and never again. From then on, MD changelogs are generated from the JSON counterparts.